packages/stylelint-config/index.js

```diff
@@ -1,47 +1,27 @@
 module.exports = {
   customSyntax: 'postcss-scss',
+  extends: [
+    "stylelint-config-standard-scss",
+    "stylelint-config-recommended"
+  ],
   rules: {
```

Comment:
Please, look at https://github.com/lmc-eu/code-quality-tools/blob/main/packages/eslint-config-react/index.js#L4 where rules are extended from `./rules` directory.

I think it will be more maintainable if we just separate `base` and `scss` rules (this should be done anyway :-)). But maybe a more detailed structure should be applied. Like `selector` rules, `declaration` rules or `function` rules. What do you think?

Reply:
Good point, this approach allows us to modify certain rules set by specific package and modify them at place. It makes it more clear and easy to read.

I've refactored it to partial rules files within the rules directory.